gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/mapper/FlowMapper.java

```diff
@@ -97,6 +98,23 @@ public io.gravitee.repository.management.model.flow.Flow toRepository(
         return repositoryFlow;
     }
 
+    public io.gravitee.repository.management.model.flow.Flow toRepositoryUpdate(
```

Comment:
As we have a FlowMapperTest.java do you think we could add a small unit test on this method ?